apps/frontend/components/FormValidation/FloodFormValidation/FloodFormValidation.tsx

```diff
@@ -187,7 +214,16 @@ export const FloodFormValidation = ({
         )}
         {isEditMode && (
           <>
-            <Button type="submit" sx={{ color: 'white', margin: 2 }}>
+            <Button
```

Comment:
use IconButton here

Reply:
Why? Iconbutton is to have an icon which is a button